docs/customization/custom-scripts.md

```diff
@@ -259,6 +259,20 @@ http://netbox/api/extras/scripts/example.MyReport/ \
 --data '{"data": {"foo": "somevalue", "bar": 123}, "commit": true}'
 ```
 
+### Via the CLI
+
+Scripts can be run on the CLI by invoking the management command:
+
+```
+python3 manage.py runscript [--commit] [--loglevel {debug,info,warning,error,critical}] --script <module>.<script> <data>
+```
+
+The required ``--script <module>.<script>`` argument is the report to run where ``<module>`` is the name of the python file in the ``scripts`` directory without the ``.py`` extension and ``<script>`` is the name of the script class in the ``<module>`` to run.
```

Comment:
`--foo` format arguments are typically optional. It would probably make more sense to make the script a positional argument (similar to `runreport`).

Reply:
Swapped them around, should be good now.